Build/libHttpClient.GDK/libHttpClient.GDK.def

```diff
@@ -108,12 +108,14 @@ EXPORTS
     HCWebSocketGetHeader
     HCWebSocketGetHeaderAtIndex
     HCWebSocketGetNumHeaders
+    HCWebSocketGetPingInterval
```

Comment:
>HCWebSocketGetPingInterval [](http://example.com/codeflow?start=4&length=26)

For both def files, add new methods to the end.  This maintains backwards compatibility for users linking with an old import lib.

Reply:
@jplafonta done, please take another look. Thanks!